fdbserver/ApplyMetadataMutation.cpp

```diff
@@ -925,7 +941,8 @@ class ApplyMetadataMutationsImpl {
 
 				if (toCommit) {
 					MutationRef privatized = m;
-					privatized.removeChecksum();
+					privatized.clearChecksumAndAccumulativeIndex();
```

Comment:
Here only accumulative checksum is set. I didn't find when checksum is reset after param2 is set.